README.md

```diff
@@ -9,6 +9,19 @@ This application is built as an example for [Oso Cloud](https://osohq.com/docs/)
 If you are looking for an example of using the Oso library, check out
 [osohq/gitclub](https://github.com/osohq/gitclub).
 
+## Dependencies
+Use a package manager such as `apt-get` (Linux distributions) or `Homebrew` (MacOS) to install the required and
+recommended software tools and packages below.
+| Software Tool/Package | Dependency | Usage |
+|-----------------------|------------|-------|
+| **npm** | *Required* | Frontend and Backend services |
+| **Oso Cloud CLI** | *Required* | Backend services |
+| **Pip** | *Required* | Backend services |
+| **PostgreSQL** | *Required* | Backend services |
+| **Python +3.10** | *Required* | Backend services |
+| **Yarn** | *Required* | Frontend |
+| **make** | *Recommended* | Automates backend services tasks and testing |
+
```

Comment:
I think it would be good to describe what we mean with the difference between:
- required, and
- recommended.

Maybe something like:
```
### Recommended Software
#### `make`
We provide a top-level `Makefile` containing build instructions. You can run the build instructions on your own, or use `make` as a convenience.
```

I think it's likely most people cloning this repository will have `make` installed - my intention with this suggestion is to clarify why we're saying `make` is not required. We could include this information in the table, as well - whatever you think is best!

Reply:
I agree, most devs will have make or will see a make file and know how to get it. In retrospect, perhaps it's simpler (and still accurate) to say make IS required. I updated the Usage section to include why it's required. No one will want to do it manually I'm sure!